ocaml/tests/test_data/repository_pkg_of_fullname_all

```diff
@@ -253,7 +253,6 @@ intel-igb-5.3.5.20-2.xs8.x86_64
 nss-pem-1.0.3-4.el7.x86_64
 gssproxy-0.7.0-17.el7.x86_64
 nss-sysinit-3.36.0-7.el7_5.x86_64
-pbis-open-8.2.3-1.7.8.xs8.x86_64
```

Comment:
why have these been removed?

Reply:
We are going to remove PBIS related packages, 
I know this is just for unittest, but remove it makes no bad.